utils/constants.tsx

```diff
@@ -1950,6 +1950,23 @@ export const InsightsCardTitles = {
     },
 };
 
+export const AsciiToUtf8Characters = {
+    a: ['à', 'á', 'â', 'ã', 'ä', 'å'],
+    e: ['è', 'é', 'ê', 'ë'],
+    i: ['ì', 'í', 'î', 'ï'],
+    o: ['ð', 'ò', 'ó', 'ô', 'õ', 'ö', 'ø'],
+    u: ['ü', 'û', 'ú', 'ù'],
+    A: ['À', 'Á', 'Â', 'Ã', 'Ä', 'Å'],
+    E: ['È', 'É', 'Ê', 'Ë'],
+    I: ['Ì', 'Í', 'Î', 'Ï'],
+    O: ['Ò', 'Ó', 'Ô', 'Õ', 'Ö'],
+    U: ['Ù', 'Ú', 'Û', 'Ü'],
+    D: ['Ð'],
+    N: ['Ñ'],
```

Comment:
Does there need to be lowercase variations of these two as well? Also, is it intentional that the uppercase versions of O and Y have different accented versions from the lowercase ones?